COMPATIBILITY.md

```diff
@@ -6,10 +6,10 @@
 The below matrix shows the compatibility of the [`opensearch-java-client`](https://search.maven.org/artifact/org.opensearch.client/opensearch-java) with versions of [`OpenSearch`](https://opensearch.org/downloads.html#opensearch).
 
 | Client Version | OpenSearch Version |
-| --- | --- |
-| 1.0.0 | 1.0.0-1.3.3 |
-| 2.0.0 | 1.3.3-2.0.1 |
-| 2.1.0 | 1.3.3-2.3.0 |
```

Comment:
Could you please add current version as well? `| 2.2.0 | 1.3.3-2.3.0 |`

Reply:
sure

Reply:
just to make sure i got this. 
i am making the table like this? because if the current version is 2.2.0 i guess i am adding a new line and stating that it is compatible with these versions.

```
| Client Version | OpenSearch Version |
|----------------|--------------------|
| 1.0.0          | 1.0.0-1.3.3        |
| 2.0.0          | 1.3.3-2.0.1        |
| 2.1.0          | 1.3.3-2.3.0        |
| 2.2.0          | 1.3.3-2.5.0        |
```

Reply:
this is confusing since there is nothing to backport really. I think I am missing something here

Reply:
Table looks good, thanks! I didn't get the `backport` part, what is it about?

Reply:
never mind. was wondering how i can see that the test is of version 2.2.0 but that is clear to me now. 
I thought this was an issue since there are failing tests. then i saw all PR's failing for unpublished opensearch versions... so nothing special here.